app/views/contexts/hub_context.py

```diff
@@ -49,10 +49,17 @@ class HubContext(Context):
     }
 
     def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
-        context = self.HUB_CONTENT_STATES[
-            "COMPLETE" if survey_complete else "INCOMPLETE"
-        ]
+        survey_status = "complete" if survey_complete else "incomplete"
+        context = self.HUB_CONTENT_STATES[survey_status]
         context["rows"] = self._get_rows(enabled_section_ids)
+        title_guidance = self._schema.get_hub_title_guidance(survey_status)
+        submission_button = self._schema.get_hub_submission_button()
+
+        if title_guidance:
+            context.update(title_guidance)
+
+        if survey_status == "complete" and submission_button:
```

Comment:
This should use `survey_complete` rather than a string comparison on `survey_status`.